src/main/java/uk/gov/ons/census/fwmt/jobservice/entity/CCSOutcomeStore.java

```diff
@@ -0,0 +1,17 @@
+package uk.gov.ons.census.fwmt.jobservice.entity;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import uk.gov.ons.census.fwmt.jobservice.service.JobCacheManager;
+
+
+@Component
+public class CCSOutcomeStore {
+
+  @Autowired
+  private JobCacheManager jobCacheManager;
```

Comment:
I dont see why we need the JobCacheStore and Impl, can we move the code in there to CCSOutcomeStore